Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -1213,11 +1217,73 @@ reportIssueOptions state =
     }
   ]
 
+sourceToDestinationConfig :: ST.HomeScreenState -> SourceToDestination.Config
+sourceToDestinationConfig state = let 
+  config = SourceToDestination.config
+  sourceToDestinationConfig' = config
+    { sourceImageConfig {
+        imageUrl = fetchImage FF_COMMON_ASSET "ny_ic_source_dot"
+      , margin = (MarginTop 3)
+      , width = V 20
+      , height = V 20
+      }
+    , sourceTextConfig {
+        text = state.data.source
+      , padding = (Padding 2 0 2 2)
+      , margin = MarginHorizontal 12 15
+      , color = Color.black800
+      , ellipsize = true
+      , maxLines = 1
+      , textStyle = Body1
+      }
+    , rideStartedAtConfig {
+        text = ""
+      , color = Color.black700
+      , visibility = GONE
+      , padding = (Padding 2 0 2 2)
+      , margin = MarginHorizontal 12 15
+      , maxLines = 1
+      , ellipsize = true
+    }
+    , destinationImageConfig {
+        imageUrl = fetchImage FF_COMMON_ASSET "ny_ic_destination"
+      , margin = (MarginTop 3)
+      , width = V 20
+      , height = V 20
+      }
+    , destinationBackground = Color.blue600
+    , destinationTextConfig {
+        text = state.data.destination
+      , padding = (Padding 2 0 2 2)
+      , margin = MarginHorizontal 12 15
+      , color = Color.greyDavy
+      , ellipsize = true
+      , maxLines = 1
+      , textStyle = Body1
+      }
+    , rideEndedAtConfig {
+        text = ""
+      , color = Color.black700
+      , visibility = GONE
```

Comment:
If this visibility is GONE then other props in rideEndedAtConfig is not required